litex_boards/targets/xilinx_zybo_z7.py

```diff
@@ -43,55 +41,226 @@ def __init__(self, platform, sys_clk_freq, use_ps7_clk=False):
 # BaseSoC ------------------------------------------------------------------------------------------
 
 class BaseSoC(SoCCore):
-    def __init__(self, sys_clk_freq=100e6, with_led_chaser=True, **kwargs):
+    def __init__(self, sys_clk_freq=int(100e6), variant="z7-10", with_ps7=False, with_led_chaser=True, **kwargs):
```

Comment:
`int()` has been dropped by commit 9e7079c4c8045092aea25fba4b396bccf2ea782b
With LiteX commit 01b9ae7894eb081258113476e58c6ac21739a11c `int()` is done at soc.py level.
`with_ps7` is not clear to me